src/Component.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\twig_fractal\Component.
+ */
+
+namespace Drupal\twig_fractal;
+
+use Drupal\Component\Utility\Html;
+use Symfony\Component\Yaml\Yaml;
+
+class Component {
```

Comment:
There should be a high-level description on each class explaining how this brick is used in the big picture.